protos/peer/chaincode_shim.proto

```diff
@@ -73,6 +75,22 @@ message GetStateMetadata {
     string collection = 2;
 }
 
+message GetStateBatch {
+    repeated string keys = 1;
+    string collection = 2;
+}
+
```

Comment:
technically you can try to read from several collections, I think structure has to reflect it.

Reply:
refactored code to be used with multiple collection